NUEntity.js

```diff
@@ -87,10 +87,10 @@ export default class NUEntity extends NUAbstractModel {
         }
         return this.mandatoryAttributes;
     }
-    
+
     static hasMandatoryAttributesSet(JSONObject) {
-        for (const attr of this.getMandatoryAttributes()) {
-            if (!JSONObject[attr]) {
+        for (const [attr, descriptor] of Object.entries(this.attributeDescriptors)) {
```

Comment:
let's still use getMandatoryAttributes! I don't want 2 different code base doing the same thing. If needed have an attribute pass to getMandataoryAttribute to get the attr object.

Reply:
@natabal I removed the getMandatoryAttributes() function altogether as it was not being used anywhere else. is this ok?